### PR TITLE
Feat/interested events UI

### PR DESCRIPTION
**Pull request by arielyxw**
Implemented UI for interested events recycler view.
Includes dummy events populating the recycler view created for testing.

To do:
Create a button in accounts page to link to the interested events view.
Firebase connection to populate with events from a database.

Partially solves #22

### CODE REVIEW

File: app/src/main/java/com/example/a310_rondayview/InterestedEventsFragment.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_interested_events, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        // load in the list of favourited events from there database here
```
Comment (boxy8) [inline app/src/main/java/com/example/a310_rondayview/InterestedEventsFragment.java]:
```suggestion
        // load in the list of favourited events from the database here
```

File: app/src/main/java/com/example/a310_rondayview/ui/InterestedEventsAdapter.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class InterestedEventsAdapter extends RecyclerView.Adapter<InterestedEventsAdapter.InterestedEventsViewHolder> {
+
+    Context context;
+    ArrayList<Event> eventsArrayList;
```
Comment (boxy8) [inline app/src/main/java/com/example/a310_rondayview/ui/InterestedEventsAdapter.java]: Does this need to be an array list? Can we change it to List<Event> so we don't have to cast it later on?

**Review by boxy8 — CHANGES_REQUESTED** [review]
LGTM just one typo that won't even matter later 🤣 and one comment

File: app/src/main/java/com/example/a310_rondayview/ui/InterestedEventsAdapter.java (hunk 2 above)
Comment (arielyxw) [reply app/src/main/java/com/example/a310_rondayview/ui/InterestedEventsAdapter.java]: Sounds like a better idea, good catch will change that now :)

**Review by boxy8 — APPROVED** [review]
LGTM!!!!! :cat:

**Pull request merged**